applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -123,7 +123,11 @@ static struct {
 	} in;
 
 	struct {
+#if CONFIG_AUDIO_BIT_DEPTH_16
 		int16_t __aligned(sizeof(uint32_t)) fifo[MAX_FIFO_SIZE];
+#elif CONFIG_AUDIO_BIT_DEPTH_32
+		int32_t __aligned(sizeof(uint32_t)) fifo[MAX_FIFO_SIZE];
```

Comment:
We probably don't need __aligned here.
But it doesn't hurt to just have it, if no other changes are needed in this PR